six/modules/python/six.sicd/tests/test_read_region.py

```diff
@@ -1,61 +0,0 @@
-#!/user/bin/env/python
```

Comment:
Since `read` and `readRegion` became the same function, which called the same thing underneath, I thought it wasn't really testing anything anymore.